cypress_test/integration_tests/mobile/calc/hamburger_menu_spec.js

```diff
@@ -182,19 +182,7 @@ describe('Trigger hamburger menu options.', function() {
 
 		mobileHelper.selectHamburgerMenuItem(['Edit', 'Cut']);
 
-		// TODO: cypress does not support clipboard operations
-		// so we get a warning dialog here.
-		cy.get('.vex-dialog-form')
-			.should('be.visible');
-
-		cy.get('.vex-dialog-message')
-			.should('have.text', 'Please use the copy/paste buttons on your on-screen keyboard.');
-
-		cy.get('.vex-dialog-buttons .button-primary')
-			.click();
-
-		cy.get('.vex-dialog-form')
-			.should('not.exist');
+		cy.get('#mobile-wizard-content').should('exist');
```

Comment:
maybe check if some unique id exists? (this only tells mobile wizard is opened. I'm not even sure if that container doesn't exist when not opened).
I think there is generated `modal-dialog-something`   container?